### PR TITLE
MyProjectsCard Bootstrap CSS

### PR DESCRIPTION
**Pull request by rohitk18**
Used Bootstrap CSS and removed table html.
Implemented for cards in MyProjects and MyGroups (uses same CSS classes)

### CODE REVIEW

File: civictechprojects/static/css/partials/_MyProjectCard.scss [hunk 1]
```diff
@@ -16,14 +16,9 @@
...
   font-weight: bold;
-}
-
-.MyProjectCard-table {
-  background-color: $color-background-light;
-  border: solid 1px $color-grey-frame-border;
-  margin: 20px auto;
-  padding: 16px;
-  width: 98%;
+  &.text-overflow-break {
```
Comment (PeterBreen) [inline civictechprojects/static/css/partials/_MyProjectCard.scss:19]: This works - but to save a few lines of CSS how about using the built-in bootstrap utility class `text-break` to do the same thing? ([ref](https://getbootstrap.com/docs/4.4/utilities/text/#word-break))

**Review by PeterBreen — CHANGES_REQUESTED** [review]
This looks really good; I have one small requested change to a utility class, beyond that no other changes needed. Thanks for taking this issue! 👍

File: civictechprojects/static/css/partials/_MyProjectCard.scss (hunk 1 above)
Comment (rohitk18) [reply civictechprojects/static/css/partials/_MyProjectCard.scss]: Replaced with the built-in bootstrap utility class `text-break`

**Comment by rohitk18** [comment]
Replaced with the built-in bootstrap utility class `text-break` in project card and group card
Removed `text-overflow-break` from _MyProjectCard.scss

**Review by PeterBreen — APPROVED** [review]
👍 Perfect, thanks!

**Pull request merged**